plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/attribute-filter.block_theme.spec.ts

```diff
@@ -97,6 +97,19 @@ test.describe( 'Product Filter: Attribute Block', () => {
 			} );
 		} );
 
+		test( 'clear button is not shown on initial page load', async ( {
+			page,
+			defaultBlockPost,
+		} ) => {
+			await page.goto( defaultBlockPost.link );
+
+			const button = page.locator(
+				'.wp-block-woocommerce-product-filter-clear-button'
+			);
```

Comment:
I will make the change but I am not sure I totally agree because what if you have multiple buttons on the page that are buttons and with the same name 'Clear'?